samples/e2eTestUtils/ConfigUtils.ts

```diff
@@ -0,0 +1,71 @@
+import fs from 'fs';
+
+export class StringReplacer {
```

Comment:
suggest you add some comments describing what this does... I think I know based on description of PR.  Are there tests for this code?

Reply:
Comments added. We currently don't test e2eUtils code -not quite sure where/when we would run them if we added so cc @tnorling

Reply:
Given this is itself only used in tests and those tests can't function without this working properly I don't think this needs special coverage. It should be very apparent if there's an issue.